src/experimental/Entry.cpp

```diff
@@ -4,7 +4,21 @@ namespace experimental {
 
 Entry::Entry(DataPosition position_)
     : position(position_)
-{}
+{
+//    position = position_;
+}
+/// [F] from what I understand the initialiser list should do exactly the same as
```

Comment:
What you gain with the initlizers is performance. Especially for our simple classes there really isnt any reason to put code in the braces because we only initilise data. One interesting use for initilizers you can find in the `DataPosition` class. Can you understand what happens? and why i do it that way?